alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/action/DeleteMonitorRequest.kt

```diff
@@ -0,0 +1,35 @@
+package com.amazon.opendistroforelasticsearch.alerting.action
+
+import org.elasticsearch.action.ActionRequest
+import org.elasticsearch.action.ActionRequestValidationException
+import org.elasticsearch.action.support.WriteRequest
+import org.elasticsearch.common.io.stream.StreamInput
+import org.elasticsearch.common.io.stream.StreamOutput
+import java.io.IOException
+
+class DeleteMonitorRequest : ActionRequest {
```

Comment:
What is the point of this class? Is it because we need the request class to extend `ActionRequest`?
Because of that, is this why we are creating a `DeleteMonitorRequest` and not a `SearchMonitorRequest` since `SearchRequest` extends `ActionRequest`?

Reply:
Yes